db/models.py

```diff
@@ -14,3 +14,35 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actor_movies")
+    genres = models.ManyToManyField(Genre, related_name="genre_movies")
```

Comment:
Related names here probably should be just a plural form of a "movie". What do you think?